### PR TITLE
Fix Python_LaserAccelerationMR on Battra

### PR DESCRIPTION
**Pull request by MaxThevenet**
The ini file for this test contains
```
outputFile = diags/plotfiles/plt00010
```
so this is required.

### CODE REVIEW

File: Examples/Physics_applications/laser_acceleration/PICMI_inputs_laser_acceleration.py [hunk 1]
```diff
@@ -102,7 +102,8 @@
 
 field_diag1 = picmi.FieldDiagnostic(name = 'diag1',
                                     grid = grid,
-                                    period = 10)
+                                    period = 1,
```
Comment (dpgrote) [inline Examples/Physics_applications/laser_acceleration/PICMI_inputs_laser_acceleration.py:105]: Thanks for making this change!

This should be `period = 10` to match the value in ParticleDiagnostic.

Comment (MaxThevenet) [inline Examples/Physics_applications/laser_acceleration/PICMI_inputs_laser_acceleration.py:105]:
```suggestion
                                    period = 10,
```

Comment (MaxThevenet) [reply Examples/Physics_applications/laser_acceleration/PICMI_inputs_laser_acceleration.py]: Oops indeed, thanks

**Comment by MaxThevenet** [comment]
This is no longer relevant as we chose a different approach in #1065.

**Pull request closed without merging**